core/src/storage/cache.py

```diff
@@ -220,10 +220,6 @@ def end_current_session() -> None:
     _active_session_idx = None
 
 
-def is_session_started() -> bool:
```

Comment:
Seems to be maybe used in tests/test_storage.cache.py?

Reply:
Right, I have defined it in a test-file in [05517d7](https://github.com/trezor/trezor-firmware/pull/2688/commits/05517d7683ce391682336bcc96ca6cf5397bd7f5)